docs/diagrams/AddState2.puml

```diff
@@ -10,29 +10,69 @@ skinparam ClassBackgroundColor #FFFFAA
 title add id/A0123456X n/John e/e0123456@u.nus.edu g/A
 
 class uniquePersonList as "<u>uniquePersonList:UniquePersonList</u>"
-class student1 as "<u>student1:Person</u>" {
+class student1 as "<u>student1:Person</u>" {}
+class student2 as "<u>student2:Person</u>" {}
+class student3 as "<u>student3:Person</u>" {}
+class studentName1 as "<u>studentName1:Name</u>" {
     name = Alice
```

Comment:
Perhaps wrap the  values with `""` since they are String values
```suggestion
    name = "Alice"
```

Similar example from the textbook:
![image](https://github.com/AY2324S2-CS2103T-T09-1/tp/assets/108446221/70806c82-985c-4a82-9783-99fe03761e6c)